src/main/java/com/sda/groupa/shippingcostcalculator/border/controller/BorderCrossController.java

```diff
@@ -0,0 +1,106 @@
+package com.sda.groupa.shippingcostcalculator.border.controller;
+
+import com.sda.groupa.shippingcostcalculator.border.model.BorderCross;
+import com.sda.groupa.shippingcostcalculator.border.model.Borders;
+import com.sda.groupa.shippingcostcalculator.border.service.BorderCrossService;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+
+@Controller
+public class BorderCrossController {
+    private BorderCrossService borderCrossService;
+
+    public BorderCrossController(BorderCrossService borderCrossService) {
+        this.borderCrossService = borderCrossService;
+    }
+
+    @GetMapping("/borderCross/add")
+    public ModelAndView getBorderCrossForm() {
+        ModelAndView modelAndView = new ModelAndView("borderCross");
+
+        BorderCross borderCross = new BorderCross();
+
+        List<Borders> borders = borderCrossService.getListOfBorders();
+
+        modelAndView.addObject("borderCross", borderCross);
+        modelAndView.addObject("listOfBorders", borders);
+
+        return modelAndView;
+    }
+
+    @GetMapping("/borders/add")
+    public ModelAndView getBorderForm() {
+        ModelAndView modelAndView = new ModelAndView("borders");
+        Borders borders = new Borders();
+        modelAndView.addObject("borders", borders);
+        return modelAndView;
+    }
+@GetMapping("/borderCross/add/{id}")
+public ModelAndView editBorderCross(@PathVariable Long id){
+
+        ModelAndView modelAndView = new ModelAndView("borderCross");
+
+    BorderCross borderCross = borderCrossService.findBorderCrossById(id).orElseThrow(() -> new RuntimeException("Unavailable"));
```

Comment:
Stwórzmy może jakiś konkretniejszy exception typu `BorderCrossNotFoundException`, który będzie rozszerzać `RuntimeException` ? Co Ty na to?